articles/extensions/logstash/index.yml

```diff
@@ -0,0 +1,8 @@
+versioning:
+  baseUrl: extensions/logstash
+  current: v2
+  versions:
+    - v1
+    - v2
```

Comment:
Why did you create a v2 of the doc, instead of updating the old one? I can see how you added two paragraphs on replay logs and slack integration. Are these two things not working for v1 of the extension?